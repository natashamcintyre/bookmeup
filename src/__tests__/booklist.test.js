import React from 'react'
import BookList from '../components/bookList'

describe('booklist', () => {
  it('renders without crashing', () => {
    const component = mount(<BookList />)
    expect(component).toMatchSnapshot()
  })

  // it('has available books list', () => {
  //   const component = mount(<BookList />);
  //   expect(component.exists('ul#books_list')).toBe(true);
  // });
})
