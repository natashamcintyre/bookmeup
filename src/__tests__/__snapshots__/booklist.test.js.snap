// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`booklist renders without crashing 1`] = `ReactWrapper {}`;
